src/Features/CSharp/Portable/ConvertToRawString/ConvertRegularStringToRawStringCodeRefactoringProvider.cs

```diff
@@ -159,6 +209,52 @@ private static bool HasLeadingWhitespace(VirtualCharSequence characters)
             return document.WithSyntaxRoot(root.ReplaceToken(token, replacement));
         }
 
+        protected override async Task FixAllAsync(
+            Document document,
+            ImmutableArray<TextSpan> fixAllSpans,
+            SyntaxEditor editor,
+            CodeActionOptionsProvider optionsProvider,
+            string? equivalenceKey,
+            CancellationToken cancellationToken)
+        {
+            Debug.Assert(equivalenceKey != null);
+
+            var options = await document.GetSyntaxFormattingOptionsAsync(optionsProvider, cancellationToken).ConfigureAwait(false);
+            using var _ = PooledDictionary<SyntaxToken, SyntaxToken>.GetInstance(out var tokenReplacementMap);
+
+            foreach (var fixSpan in fixAllSpans)
+            {
+                var node = editor.OriginalRoot.FindNode(fixSpan);
+                foreach (var stringLiteral in node.DescendantTokens(descendIntoTrivia: true).Where(token => token.Kind() == SyntaxKind.StringLiteralToken))
+                {
+                    // Ensure we can convert the string literal
+                    if (!CanConvertStringLiteral(stringLiteral, out var canConvertParams))
+                        continue;
+
+                    // Get the kind to be fixed from the equivalenceKey for the FixAll operation
+                    var kind = s_kindToEquivalenceKeyMap.GetKeyOrDefault(equivalenceKey);
+
+                    // Ensure we have a matching kind to fix for this literal
+                    var hasMatchingKind = kind switch
+                    {
+                        ConvertToRawKind.SingleLine => canConvertParams.CanBeSingleLine,
+                        ConvertToRawKind.MultiLineIndented => !canConvertParams.CanBeSingleLine,
+                        ConvertToRawKind.MultiLineWithoutLeadingWhitespace => canConvertParams.CanBeMultiLineWithoutLeadingWhiteSpaces,
+                        _ => throw ExceptionUtilities.UnexpectedValue(kind),
+                    };
+
+                    if (!hasMatchingKind)
+                        continue;
+
+                    var replacement = GetReplacementToken(document, stringLiteral, kind, options, cancellationToken);
+                    tokenReplacementMap.Add(stringLiteral, replacement);
```

Comment:
do we need teh replacement map?  Could just use the editor directly right and do `.ReplaceNode` on it with the token's parent?

Reply:
I thought this will be more efficient. Unless you see any disadvantage, I'll keep the current approach.